core/src/cluster_info_vote_listener.rs

```diff
@@ -389,17 +409,14 @@ impl ClusterInfoVoteListener {
             let would_be_leader = poh_recorder
                 .lock()
                 .unwrap()
-                .would_be_leader(20 * DEFAULT_TICKS_PER_SLOT);
+                .would_be_leader(3 * slot_hashes::MAX_ENTRIES as u64 * DEFAULT_TICKS_PER_SLOT);
```

Comment:
this changes from `20` to `3 * MAX_ENTRIES = 3 * 512 = 1536`, just want to double-check is intended.

Reply:
yeah, as mentioned in the PR description the window was a bit short. Seeing as we only keep around `MAX_ENTRIES` votes anyways, this allows for about 3 forks of size `MAX_ENTRIES` each